pom.xml

```diff
@@ -21,7 +21,7 @@
         <air.check.skip-license>true</air.check.skip-license>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.javadoc.lint>-missing</air.javadoc.lint>
-        <project.build.targetJdk>11</project.build.targetJdk>
+        <project.build.targetJdk>17</project.build.targetJdk>
```

Comment:
```
trino master$ mvnd dependency:tree -pl :trino-jdbc | grep units
[INFO] +- io.airlift:units:jar:1.7:compile
```

but trino-jdbc is still on 8, while we already bumped here to 11 in https://github.com/airlift/units/pull/29

Reply:
Yes, `trino-jdbc` is pinned to the last version that was targeting JDK 8. Units 1.8 was targeting 11, so it can no longer be used in `trino-jdbc`.